content/en/cloudcraft/getting-started/datadog-integration.md

```diff
@@ -0,0 +1,63 @@
+
+By using Datadog SSO, you automatically gain access to:
+
+- **Cross-platform functionality**: Move effortlessly between Cloudcraft and Datadog to analyze your cloud infrastructure and its performance.
+- **Automated cloud account synchronization**: Cloud accounts configured in Datadog sync automatically with Cloudcraft, giving you a complete view of your infrastructure in both platforms.
+
+### Enable Datadog SSO for existing accounts
+
+If you originally signed up with a different login method, for example, Google SSO or a standard username and password, you won’t have access to the full set of Datadog integration features. To switch to Datadog SSO, [contact the Cloudcraft support team][1] and they will assist you in converting your account.
+
+## Pull cloud accounts
+
+<div class="alert alert-info">This feature supports only Amazon Web Services (AWS) accounts. Syncing with Azure or other cloud providers is not available at this time.</div>
+
+The integration between Cloudcraft and Datadog streamlines cloud account management, allowing accounts already configured in Datadog to be automatically pulled into Cloudcraft. No additional setup is required in Cloudcraft as long as resource collection is enabled in Datadog.
+
+### How it works
+
+1. **Pulling cloud accounts**: Once your AWS account is integrated with Datadog, Cloudcraft automatically pulls the account, so you don’t need to manually add it to Cloudcraft.
+2. **Resource Collection in Datadog**: To visualize and diagram resources in Cloudcraft, ensure resource collection is enabled in Datadog. Without this feature, your AWS accounts will still be pulled into Cloudcraft, but no resources from them will be available for diagramming. Refer to the [AWS integration documentation][2] for steps on enabling resource collection.
+3. **No AWS accounts in Datadog?**: If you don't have any AWS accounts added in Datadog, you'll need to add them first. Follow the instructions in the [AWS integration guide][3].
+
+### Manage pulled AWS accounts in Cloudcraft
+
+AWS accounts pulled from Datadog are marked with the Bits icon in the account selector under the **Live** tab in Cloudcraft. These accounts are shared with the entire team by default.
+
+To manage the visibility settings of these accounts:
+
+1. Go to **User > AWS Accounts**.
+2. Select the **Edit** icon (the pencil icon next to the account name).
+3. Toggle the **Visibility on Live** tab option to control whether the account is visible to the team.
+
+### Performance benefits
+
+AWS accounts pulled from Datadog offer improved performance when creating diagrams in Cloudcraft compared to accounts added directly in Cloudcraft, as Cloudcraft uses Datadog APIs, which are optimized for faster performance, instead of AWS APIs.
```

Comment:
This is not true reason - the reason why it is faster because the data is already collected and stored by Datadog (so it is not real time but ~ 15 min old.